signalfx-agent/meta.yaml

```diff
@@ -1,5 +1,5 @@
 code: https://github.com/signalfx/signalfx-agent
-data_signature: _exists_:signalfx_agent
+data_signature: plugin:host-metadata
```

Comment:
Does the value `host-metadata` need quotes like the one in the old collectd tile?  Not sure if the `-` causes problems or not.